scripts/nb-tester/test-notebook.py

```diff
@@ -14,78 +14,149 @@
 from pathlib import Path
 import nbconvert
 import nbformat
+import nbclient
+from datetime import datetime
+from qiskit_ibm_runtime import QiskitRuntimeService
+import argparse
 
-WRITE_FLAG = "--write"
 NOTEBOOKS_GLOB = "docs/**/*.ipynb"
 NOTEBOOKS_EXCLUDE = [
     "docs/api/**",
     "**/.ipynb_checkpoints/**",
-    # Following notebooks have code errors
+    # Following notebooks are broken
     "docs/transpile/transpiler-stages.ipynb",
-    # Following notebooks make requests so can't be tested yet
     "docs/run/get-backend-information.ipynb",
```

Comment:
This notebook can now be tested, but is broken as `ibm_lagos` doesn't exist any more. I'll fix in a follow-up.